.env

```diff
@@ -0,0 +1,3 @@
+# Sets the python source roots in vscode to match Pants' repo layout
+
```

Comment:
Should probably mention this can be removed whenever https://github.com/pantsbuild/pants/issues/10920 "fixes' this issue.

Although this issue isn't VSCode specific.

Reply:
@joshua-cannon-techlabs see updated